source/features/set-default-repositories-type-to-sources.tsx

```diff
@@ -6,7 +6,7 @@ function init(): void {
 	const links = select.all<HTMLAnchorElement>([
 		'.user-profile-nav [href$="tab=repositories"]', // "Your repositories" in header dropdown
 		'#user-links [href$="tab=repositories"]', // "Repositories" tab on user profile
-		'.orgnav .pagehead-tabs-item:first-child', // "Repositories" tab on organization profile
+		'.orgnav .pagehead-tabs-item:first-child:not([href$="/settings/profile"])', // "Repositories" tab on organization profile and exclude the settings tab
```

Comment:
Example of [organization page](https://github.com/babel) without a Repositories tab?

Reply:
@fregante no it catches both the repo tab and the settings tab

Reply:
`.orgnav` does not appear on repo pages.

Reply:
I don't get it. `.orgnav .pagehead-tabs-item:first-child` must always be the repositories tab. 

Example URL.

Reply:
@fregante I'll be by a computer in about 20 min

Reply:
Go to a org that you have access to
![image](https://user-images.githubusercontent.com/16872793/82218147-75d64500-98e9-11ea-9e70-a9a831c53bee.png)

The second one is the settings tab, I tested on https://github.com/talknsmile

Reply:
```suggestion
		'.orgnav > .pagehead-tabs-item:first-child', // "Repositories" tab on organization profile
```